hanko-js/src/ui/pages/RegisterPassword.tsx

```diff
@@ -76,6 +76,8 @@ const RegisterPassword = ({ user, registerAuthenticator }: Props) => {
         <InputText
           type={"password"}
           name={"password"}
+          autocomplete={"new-password"}
```

Comment:
There is no autocompletion on my end. Tested with Safari and Chrome.

Reply:
Same as above